comm/src/test/scala/coop/rchain/comm/transport/StreamHandlerSpec.scala

```diff
@@ -76,13 +79,32 @@ class StreamHandlerSpec extends FunSpec with Matchers with BeforeAndAfterEach {
       // then
       msg.path.getParent shouldBe nonExistingWithPersmission
     }
+
+    it("should stop receiving a stream if will not fit in memory") {
+      // given
+      val alwaysBreak: CircuitBreaker = kp(true)
+      val stream                      = createStream()
```

Comment:
Perhaps it would make sense to test for a handful of generated streams

Reply:
👍

Reply:
I've changed that not with property test, but with a bit more complex unit test